docs/developing_iree/testing_guide.md

```diff
@@ -192,7 +192,7 @@ drivers).
 With CMake, run this from the build directory:
 
 ```shell
-$ ctest -R iree/test/e2e/xla_ops/check_vmla_vmla_floor.mlir
+$ ctest -R iree/test/e2e/xla_ops:check_vmla_vmla_floor.mlir
```

Comment:
can separately look at getting those `:`s changed into `/` to match our other tests:

![image](https://user-images.githubusercontent.com/4010439/97506135-77e00d80-1937-11eb-8da2-8b01ccd1c2f6.png)